packages/nextjs-kit/src/components/contentWithImage.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Image from 'next/image';
+import type { ImageProps } from 'next/image';
+import { useRouter } from 'next/router';
+
+interface ContentProps {
+	title: string;
+	imageProps?: ImageProps['src'];
+	content: string;
+	date?: string;
```

Comment:
Small thing, but let's put the optional props at the bottom of the interface.
Also we will want to add `imageAltText` as a prop as well, and we should fall back to the `title` if we don't get alt text from the user.